VideoLocker/src/main/java/org/edx/mobile/view/adapters/DiscussionTopicsAdapter.java

```diff
@@ -6,41 +6,38 @@
 import android.widget.TextView;
 
 import com.google.inject.Inject;
-import org.edx.mobile.discussion.DiscussionTopicDepth;
 
 import org.edx.mobile.R;
 import org.edx.mobile.core.IEdxEnvironment;
+import org.edx.mobile.discussion.DiscussionTopicDepth;
 
 public class DiscussionTopicsAdapter extends BaseListAdapter<DiscussionTopicDepth> {
 
-    private static final int CHILD_INDENT_LEVEL = 100;
+    private final int childPadding;
 
     @Inject
     public DiscussionTopicsAdapter(Context context, IEdxEnvironment environment) {
         super(context, R.layout.row_discussion_topic, environment);
+        childPadding = context.getResources().getDimensionPixelOffset(R.dimen.edx_margin);
     }
 
     @Override
     public void render(BaseViewHolder tag, DiscussionTopicDepth discussionTopic) {
         ViewHolder holder = (ViewHolder) tag;
-
-        String topicName = discussionTopic.getDiscussionTopic().getName();
-        holder.discussionTopicNameTextView.setText(topicName);
-
-        int childIndentLevel = discussionTopic.getDepth() * CHILD_INDENT_LEVEL;
-        holder.discussionTopicNameTextView.setPadding(childIndentLevel == 0 ? 30 : childIndentLevel, 0, 0, 0);
+        holder.discussionTopicNameTextView.setText(discussionTopic.getDiscussionTopic().getName());
+        holder.discussionTopicNameTextView.setPadding(childPadding * (1 + discussionTopic.getDepth()), childPadding, childPadding, childPadding);
```

Comment:
We should probably use relative padding here in order to support RTL mode, if allowed in the API level.